wiki/Glossary/en.md

```diff
@@ -14,12 +14,12 @@ When a thread is or is said to be "wastelanded", it is moved there.
 ### AFK
 
 AFK is an internet acronym that means _Away From Keyboard_.
-An example of this is when an active user walks off to do something else while they're still logged on.
+An example of this is when an active user walks off to do something else while they are still logged on.
 _osu!_ will mark users as "Idle" if they have been inactive for some time.
```

Comment:
the in-game card says:
> Idle
see https://osu.ppy.sh/help/wiki/Chat_Console/Chat_Console-Idle.png

... unless that is out of date?